Examples/Examples-iOS/IGListKitExamples.xcodeproj/project.pbxproj

```diff
@@ -1062,7 +1062,7 @@
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				DEVELOPMENT_TEAM = "";
 				INFOPLIST_FILE = IGListKitExamples/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
```

Comment:
Why change this? We ship to iOS 8, so supporting min 8 in the examples seems ok to me.

Reply:
I agree. I wasn't sure if we need to change it to iOS 10. But when I compile the example project in Xcode 9 beta 6, it gives me compile error. I can revert this change back if you want.